pkg/sloop/processing/eventcount.go

```diff
@@ -69,6 +70,15 @@ func updateEventCountTable(
 
 	eventCountByMinute := spreadOutEvents(computedFirstTs, computedLastTs, computedCount)
 
+	if involvedObject.Uid == "" {
+		glog.V(common.GlogVerbose).Infof("Got empty Uid for name: %v, namespace: %v, kind: %v for event: %v ", involvedObject.Name, involvedObject.Namespace, involvedObject.Kind, newEventInfo.Reason)
+		returnedUid, err := GetUidForWatchEntry(tables, txn, involvedObject.Kind, involvedObject.Namespace, involvedObject.Name, time.Time{})
+		if err != nil {
+			return err
```

Comment:
errors.Wrap(err, "Error retrieving Uid for the Watch Entry") ?

Reply:
sure! updated